.github/workflows/hz-enterprise-op.-rhel-release.yaml

```diff
@@ -375,3 +376,20 @@ jobs:
           gh pr create \
               --title "Update versions after release of RHEL ${OPERATOR_IMAGE}" \
               --body "Released ${OPERATOR_IMAGE} on RHEL. New operator and operator-bundle images are released on RHEL registry. "
+
+
+  slack_notify:
+    name: Slack Notify
+    needs: build_publish 
+    runs-on: ubuntu-latest
+    if: always()
```

Comment:
Do we want to get the notification always or only if the pipeline fails? WDYT? @hasancelik any opinion on that?

Reply:
Actually this statement is only there to run the Slack job if the first one doesn't succeed. In the Slack job there is only one step and it has the following condition `if: needs.build_publish.result != 'success'`. So, we will only get notifications when the pipeline fails. I tried to write the condition at job scope, but it didn't work so I did this workaround.